packages/expo-modules-core/ios/Swift/ExpoBridgeModule.swift

```diff
@@ -41,6 +41,30 @@ public final class ExpoBridgeModule: NSObject, RCTBridgeModule {
   public var bridge: RCTBridge! {
     didSet {
       appContext.reactBridge = bridge
+
+      if bridge.responds(to: Selector(("runtime"))) {
```

Comment:
Not too much parentheses? 😂 

```suggestion
      if bridge.responds(to: Selector("runtime")) {
```

Reply:
Actually, that was intentional to suppress the XCode warning: 
![image](https://user-images.githubusercontent.com/9578601/216330451-a6018c8f-7340-4337-99c5-4c49225d0079.png)

Reply:
Ah ok, I didn't know this suppresses that warning 🙂